### DISCUSSION TITLE
Specified folder as a source

### DISCUSSION THREAD

**Discussion by doctrees** [RafaelGB/obsidian-db-folder#937] [Ideas]
I would find it very helpful if I could select a specific folder as a source in the same way "Select destination folder" for new entries is set.
Reason is that my workflow is such that I have folders and databases like:
- Folder1
- Folder2
- Folder3
- Database for Folder1
- Database for Folder2
- Database for Folder3

I don't want to have the databases inside those folder for organizational reasons. Currently I'm using Dataview query, but quite often when I switch to different tab and back there will be some random entries/files displayed in the table, sometimes files I have created outside the searched folder. I have no idea if that's a Dataview bug or DB Folder bug.

In any case, manually selecting destination folder would fix the issue but also I guess it's simpler and faster way than going through Dataview query?

**Comment by alex-mos** [comment]
@doctrees Here is a solution: https://youtu.be/lIuZAik1jPM?t=389
